app/controllers/admin/users_controller.rb

```diff
@@ -45,5 +46,24 @@ def user_params
                                    :master_of_ceremonies,
                                    :judge
     end
+
+    def catch_lock_scores
+      lock_scores = params.fetch(:user, {}).permit(:lock_scores)
+      return if current_user.id != params.require(:id).to_i || lock_scores.blank?
+
+      @user.update lock_scores
+      notice = case lock_scores[:lock_scores]
+               when 'true'
+                 'Scores have been locked.'
+               when 'false'
+                 if User.scoring_enabled?
+                   'Scores have been unlocked.'
+                 else
+                   'Scores are still locked by other users!'
+                 end
+               end
+
+      redirect_to scoreboard_participants_path, notice:
```

Comment:
stupid ruby 3.1 I was gonna comment "silly louis you forgot something" but no, it is I who has been silly